internal/node/node.bzl

```diff
@@ -283,7 +283,11 @@ fi
             fail("""nodejs_binary/nodejs_test only support chdir inside the current package
                     but %s is not a subfolder of %s""" % (ctx.attr.chdir, ctx.label.package))
         chdir_script = ctx.actions.declare_file(_join(relative_dir, "__chdir.js__"))
-        ctx.actions.write(chdir_script, "process.chdir(__dirname)")
+        ctx.actions.write(chdir_script, """
```

Comment:
Add a comment so we understand why you bother to check, or add some regression test.

Otherwise someone will read this code later and think the conditional is unneeded because it should be a no-op to chdir to the directory which is already the current working dir.